public/app/core/components/sidemenu/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { getConfig } from 'app/core/config';
+
+export const getForcedLoginUrl = (url: string) => {
+  return `${getConfig().appSubUrl}${url}&forceLogin=true`;
```

Comment:
This can be a problem if the url does not contain any other query parameters eg. if the `url` is `http://localhost:3000/grafana/dashboards` the signin url wil become: `http://localhost:3000/grafana/dashboards&forceLogin=true`.

An alternative (but ugly approach) for solving this is [here](https://github.com/grafana/grafana/pull/25567/commits/ed6d2216a3a8b3e4dab25f455aa6ebf49fc6005e#diff-4fe11550fd8c741c221f27c3e5db5737R7).

Reply:
you are absolutely right, missed this case :/

Reply:
Fixed it in a simpler way in 4d4937d @papagian